chains/evm/cli/deploy/deploy.go

```diff
@@ -180,15 +199,40 @@ func DeployCLI(cmd *cobra.Command, args []string, txFabric calls.TxFabric, gasPr
 				return err
 			}
 			deployedContracts["erc20Token"] = erc20Addr.String()
+
+		case "erc721":
+			log.Debug().Msgf("deploying ERC721..")
+
+			if Erc721Name == "" || Erc721Symbol == "" {
+				log.Error().Err(ErrErc20TokenAndSymbolNotProvided)
+				return ErrErc20TokenAndSymbolNotProvided
+			}
+
+			erc721Addr, err := calls.DeployErc721(ethClient, txFabric, gasPricer, Erc721Name, Erc721Symbol, Erc721BaseURI)
 			if err != nil {
-				log.Error().Err(err)
+				log.Error().Err(fmt.Errorf("ERC721 deploy failed: %w", err))
 				return err
 			}
-			if Erc20Name == "" || Erc20Symbol == "" {
-				log.Error().Err(ErrErc20TokenAndSymbolNotProvided)
-				return ErrErc20TokenAndSymbolNotProvided
+
+			deployedContracts["erc721Token"] = erc721Addr.String()
+
+		case "erc721Handler":
+			log.Debug().Msgf("deploying ERC721 handler..")
+
+			emptyAddress := common.Address{}
+			if bridgeAddr == emptyAddress {
```

Comment:
Might as well have it just if bridgeAddr = common.Address{} if it is not used anywhere else.

Reply:
This is something that was just waiting for Tihomirs changes in `deploy.go` to be merged as it is obsolete code with new CLI code structure.